css/main.css

```diff
@@ -325,3 +325,86 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.mu-contact-bottom {
+  display: inline;
+  float: left;
+  width: 100%;
+}
+
+#form-messages {
+  margin-bottom: 10px;
+  background-color: #fff;
+}
+
+.mu-contact-form .form-group input {
+  background: transparent;
+  border-radius: 8px;
+  color: #333;
+  font-size: 15px;
+  border: 1px solid #888;
+  height: 45px;
+  margin-bottom: 24px;
+}
```

Comment:
- [ ] *[Required]* The border color should be white (#fff)
- [ ] *[Required]* According to the design, there should be no border radius
- [ ] *[Required]* If you follow the design, you'll notice that the only border here should be border bottom

Reply:
Done!